Lib/test/test_gdb.py

```diff
@@ -55,7 +55,7 @@ def get_gdb_version():
 if not sysconfig.is_python_build():
     raise unittest.SkipTest("test_gdb only works on source builds at the moment.")
 
-if 'Clang' in platform.python_compiler() and sys.platform == 'darwin':
+if 'Clang' in platform.python_compiler():
```

Comment:
Would it not make more sense to skip if Python is _not built with GCC_? Python can for example be built with `icc`.

Reply:
In case of doubt, I prefer to not skip the test. Maybe icc is just fine.